### PR TITLE
feat:added user feed

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
This is small PR, code quality looks good 👍

File: src/components/CreatePost.jsx [hunk 1]
```diff
@@ -3,20 +3,22 @@ import { useState } from "react"
...
-    const { setIsCreatePostVis, dispatchPost } = methods
+const CreatePost = ({ props }) => {
+    const { postsState, setIsCreatePostVis, dispatchPost } = props
     const [postContent, setPostContent] = useState('')
 
+    const createPostHandler = async (postContent) => {
+        setIsCreatePostVis(false)
+        const res = await createPostApiCall(postContent);
+        dispatchPost({ type: 'UPDATE', payload: { ...postsState, allPosts: res.data.posts } })
+        console.log(res.data)
```
Comment (nikhiljugale007) [inline src/components/CreatePost.jsx:14]: remove console log

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
+
+    const likeHandler = async (post) => {
+        const res = await likePostApiCall(post)
+        dispatchPost({ type: 'UPDATE', payload: { ...postsState, allPosts: res.data.posts } })
+        console.log(res.data.posts, 'after likinf')
+    }
+
+    const deleteHandler = async (post) => {
+        const res = await deletePostApiCall(post)
+        dispatchPost({ type: 'UPDATE', payload: { ...postsState, allPosts: res.data.posts } })
```
Comment (nikhiljugale007) [inline src/pages/Home.jsx:20]: action.type can be renamed as POST_UPDATE for better readability.

**Review by nikhiljugale007 — COMMENTED** [review]
UI suggestions:
When selected trending the UI breaks, check that.
Add hover effects on like and comment button.

**Pull request merged**